sdk/storage/azure-storage-blob/tests/test_retry.py

```diff
@@ -32,6 +32,16 @@
     GlobalStorageAccountPreparer
 )
 
+class RetryRequestTransport(RequestsTransport):
+    """Transport to test retry"""
+    def __init__(self, *args, **kwargs):
+        self.count = 0
+        super().__init__(*args, **kwargs)
+    
+    def send(self, request, **kwargs):
+        self.count += 1
+        response = super().send(request, **kwargs)
```

Comment:
Unfortunately on Pyhon 2 tests, you need to long "super" declaration....

Reply:
pushed